samples/json/ip/ipblockputtag.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name" : "ip block tag name",
```

Comment:
These should be lists to reflect the request

Reply:
It's still an object, you just set a `tags` key property - shouldn't it be a list?